packages/world/src/utils/InventoryUtils.sol

```diff
@@ -40,46 +40,52 @@ struct ToolData {
   ObjectType toolType;
   uint16 slot;
   uint128 massLeft;
-  uint128 maxUseMass;
 }
 
 library InventoryUtils {
   function getToolData(EntityId owner, uint16 slot) public view returns (ToolData memory) {
     EntityId tool = InventorySlot._getEntityId(owner, slot);
     if (!tool.exists()) {
-      return ToolData(owner, tool, ObjectTypes.Null, slot, 0, 0);
+      return ToolData(owner, tool, ObjectTypes.Null, slot, 0);
     }
 
     ObjectType toolType = EntityObjectType._get(tool);
     require(toolType.isTool(), "Inventory item is not a tool");
 
-    uint128 maxMass = ObjectPhysics._getMass(toolType);
-    uint128 maxUsePerCall = maxMass / 10; // Limit to 10% of max mass per use
-
-    return ToolData(owner, tool, toolType, slot, Mass._getMass(tool), maxUsePerCall);
+    return ToolData(owner, tool, toolType, slot, Mass._getMass(tool));
   }
 
-  function useTool(EntityId owner, Vec3 ownerCoord, uint16 slot, uint128 useMassMax)
-    public
-    returns (ObjectType toolType)
-  {
+  function useTool(EntityId owner, uint16 slot, uint128 useMassMax) public returns (ObjectType toolType) {
     ToolData memory toolData = getToolData(owner, slot);
 
-    uint128 massReduction = toolData.getMassReduction(useMassMax);
-    applyMassReduction(toolData, ownerCoord, massReduction);
+    (, uint128 toolMassReduction) = toolData.getMassReduction(useMassMax, 1);
```

Comment:
why do we hardcode a multiplier here? or I guess where is this used? since for mines and hit we dont call this